vxquery-core/src/main/java/org/apache/vxquery/common/VXQueryCommons.java

```diff
@@ -0,0 +1,52 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.apache.vxquery.common;
+
+import org.apache.hyracks.algebricks.core.algebra.functions.FunctionIdentifier;
+import org.apache.vxquery.functions.BuiltinFunctions;
+import org.apache.vxquery.runtime.functions.index.indexCentralizer.IndexCentralizerUtil;
+import org.xml.sax.SAXException;
+
+import javax.xml.bind.JAXBException;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+public class VXQueryCommons {
+
+    public static final Set<FunctionIdentifier> collectionFunctions = new HashSet<>();
+
+    public static final Set<FunctionIdentifier> indexingFunctions = new HashSet<>();
+
+    public static IndexCentralizerUtil INDEX_CENTRALIZER_UTIL = null;
+    static {
+        INDEX_CENTRALIZER_UTIL = new IndexCentralizerUtil();
+        INDEX_CENTRALIZER_UTIL.readIndexDirectory();
+
+    }
+
+    static {
```

Comment:
Split this static assignment into two parts. Add each part below the variable its assigning values to. Similar to INDEX_CENTRALIZER_UTIL.